<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgdelete_update</class>
 <widget class="QDialog" name="dlgdelete_update">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>名称：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="leName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>类型：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QComboBox" name="cbType">
       <item>
        <property name="text">
         <string>盖饭</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>盒饭</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>菜系</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>主食</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>其它</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>价格：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="lePrice"/>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>元</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>热度值：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="leHotLevel"/>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pbDelete">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QPushButton" name="pbUpdate">
       <property name="text">
        <string>更新</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>118</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="QPushButton" name="pbExit">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
